src/main/scala/com/microsoft/partnercatalyst/fortis/spark/ProjectFortis.scala

```diff
@@ -168,6 +170,19 @@ object ProjectFortis extends App {
               "consumerGroup" -> "$Default"
             )
           )
+        ),
+        "customevents" -> List(
+          ConnectorConfig(
+            "CustomEvents",
+            Map (
+              "policyName" -> envOrElse("CUSTOMEVENTS_EH_POLICY_NAME", "project-fortis-spark"),
```

Comment:
Can't you fetch the shared access policy and get the details from there? It's all exposed in the Azure portal ([deep link](https://ms.portal.azure.com/#resource/subscriptions/04f7ec88-8e28-41ed-8537-5e17766001f5/resourceGroups/fortis-resource-group/providers/Microsoft.EventHub/namespaces/fortiscustomevents/saskey)).

![image](https://user-images.githubusercontent.com/1086421/27655889-d855ee3e-5bfb-11e7-84f6-8ff6539a948c.png)